README.md

```diff
@@ -171,7 +171,7 @@ A collection of awesome things regarding the React ecosystem.
 - [react-cosmos](https://github.com/react-cosmos/react-cosmos) - Dev tool for creating reusable React components
 - [eslint-plugin-react](https://github.com/yannickcr/eslint-plugin-react) - React specific linting rules for ESLint
 - [eslint-plugin-jsx-a11y](https://github.com/evcohen/eslint-plugin-jsx-a11y) - Static AST checker for a11y rules on JSX elements
-- [react-axe](https://github.com/dequelabs/react-axe) - Accessibility auditing for React applications
+- [@axe-core/react](https://github.com/dequelabs/axe-core-npm) - React accessibility package available via NPM as @axe-core/react.
```

Comment:
```suggestion
- [@axe-core/react](https://github.com/dequelabs/axe-core-npm) - Accessibility auditing for React applications
```

I think the previous explanation was better

Reply:
Agreed.